docusaurus/docs-internals/editor-controls/data-controls/number.md

```diff
@@ -0,0 +1,53 @@
+|     `position`      | number  |    -    | The position of the control in toolbar                                                                                                                                                                                                                            |
+|      `devices`      | string  | `"all"` | Specify the devices on which the control will be displayed. Values: `"all"`, `"desktop"`, `"responsive"`. `"all"` will display control on all devices. `"desktop"` will display control only on desktop. `"responsive"` will display control on tablet and mobile |
+|     `disabled`      | boolean |  false  | Configure the condition under which the control is disabled or enabled.                                                                                                                                                                                           |
+|    `config.min`     | number  |    0    | The minimum number                                                                                                                                                                                                                                                |
+|    `config.max`     | number  |   100   | The maximum number                                                                                                                                                                                                                                                |
+|    `config.step`    | number  |    1    | The intervals value that will be incremented or decremented when using the controls’ spinners.                                                                                                                                                                    |
+|  `config.spinner`   | boolean |  true   | Enables or disables the display of arrows that increment or decrement a number                                                                                                                                                                                    |
+| `config.updateRate` | number  |   16    | The rate at which the component updates its value, specified in milliseconds. This can control how frequently changes to the input value are propagated                                                                                                           |
+|    `config.size`    | string  | "short" | A string that specifies the size of the input field. Common values might include: "short","medium","large","auto".                                                                                                                                                |
+
+### Return value
+
+The return value is "number".
+
+### Usage
+
+```js
+{
+  id: "loop",
+  label: "Loop",
+  type: "number,
```

Comment:
close the double quote

![image](https://github.com/EasyBrizy/Brizy-Local-Editor/assets/52622998/ffd82c79-39db-4335-ab77-feef8a5daba5)